__tests__/buildJsonResults.test.js

```diff
@@ -102,6 +102,10 @@ describe('buildJsonResults', () => {
   it('should support displayName template var for jest multi-project', () => {
     const multiProjectNoFailingTestsReport = require('../__mocks__/multi-project-no-failing-tests.json');
 
+    // Mock Date.now() to return a fixed later value
+    const startDate = new Date(multiProjectNoFailingTestsReport.startTime);
+    spyOn(Date, 'now').and.returnValue(startDate.getTime() + 1234);
```

Comment:
Should use `jest.spyOn` - `spyOn` global will stop working whenever we fully decouple from jasmine

Reply:
Good call - I'll fix this shortly :)

Reply:
@SimenB Oh my, I wasn't aware of that. 😕Is that deprecation documented somewhere? Couldn't find any mentions about it on neither https://facebook.github.io/jest/ or https://github.com/facebook/jest

Reply:
You won't find `spyOn` documented anywhere, closest I've got. 🙂 It just exists for legacy purposes (and because of the underlying Jasmine implementation), although there are no concrete plans for removal at this time

Reply:
Ah, didn't realize that, I'll keep an eye out for this in the future. 👍Thanks for the heads up!